client/overview/index.js

```diff
@@ -201,7 +216,10 @@ const OverviewPage = () => {
 		} else {
 			// This is the case where we addressed everything and previously had some notifications to address.
 			// We recommend the merchant to reload the page in this case.
-			if ( stripeNotificationsCountToAddressMemo > 0 ) {
+			if (
```

Comment:
I managed to get the Refresh message to show once, but I still need to investigate why the component preloader is displayed twice on refresh.